lua/lang/rust.lua

```diff
@@ -41,7 +42,10 @@ end
 
 M.lint = function()
   -- TODO: implement linters (if applicable)
-  return "No linters configured!"
+  require("lint).linters_by_fit =  {
```

Comment:
closing quote missing, `require("lint")` maybe?

Reply:
Ah I was looking after that missing quote! Thanks xD